lib/taste_tester/commands.rb

```diff
@@ -109,6 +109,10 @@ def self.test
           tested_hosts << hostname
         rescue TasteTester::Exceptions::AlreadyTestingError => e
           logger.error("User #{e.username} is already testing on #{hostname}")
+        rescue => e
```

Comment:
In this case I see why you'd do this... in general it's best practice to rescue `StandardError` which nearly everything is inherited from instead of nothing, and we should start with that...